source/configuration/modules/imfile.rst

```diff
@@ -605,8 +601,8 @@ Legacy Configuration Directives
 
    Equivalent to: Ruleset
 
-Legacy Example
-^^^^^^^^^^^^^^
```

Comment:
I think it is too early now, but  in the end we should also remove obsolete examples. We need new style instead ;-)

Reply:
That makes sense. I'll open a new issue soon and will try to include some targeted questions to make sure I have the right focus as I modify the docs going forward.